src/components/application_manager/test/commands/command_request_impl_test.cc

```diff
@@ -156,12 +156,12 @@ TEST_F(CommandRequestImplTest, OnTimeOut_StateAwaitingHMIResponse_SUCCESS) {
 
   CommandPtr command = CreateCommand<UCommandRequestImpl>(msg);
 
-  // Should be called twice:
-  // First -- on `onTimeOut` method call
-  // Second -- on destruction;
-  EXPECT_CALL(event_dispatcher_, remove_observer(_)).Times(2);
-  EXPECT_CALL(app_mngr_,
-              ManageMobileCommand(_, Command::CommandOrigin::ORIGIN_SDL));
+  MessageSharedPtr dummy_msg(CreateMessage());
+  EXPECT_CALL(*am::MockMessageHelper::message_helper_mock(),
+              CreateNegativeResponse(_, _, _, _)).WillOnce(Return(dummy_msg));
```

Comment:
@LevchenkoS can do something like that **message_helper_mock_ =  am::MockMessageHelper::message_helper_mock();**  to avoid calling **am::MockMessageHelper::message_helper_mock()** in EXPECT_CALL